apps/mail/components/create/mention-list.tsx

```
@@ -0,0 +1,115 @@
+          key={item.id}
+          className={cn(
+            'focus:bg-accent focus:text-accent-foreground relative flex cursor-default select-none items-center gap-2 rounded-sm px-2 py-1.5 text-sm outline-none transition-colors data-[disabled]:pointer-events-none data-[disabled]:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0',
+            index === selectedIndex && 'bg-accent text-accent-foreground'
+          )}
+          onClick={() => selectItem(index)}
+        >
+          <div className="flex h-6 w-6 items-center justify-center flex-shrink-0">
+            {item.logo ? (
+              <div 
+                className="w-4 h-4 text-muted-foreground" 
+                dangerouslySetInnerHTML={{ __html: item.logo }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)